src/components/BlockchainList/PostsList.tsx

```diff
@@ -1,36 +1,62 @@
-import { Suspense } from 'preact/compat'
+import { LoadingText } from 'components/Text'
+import { Suspense, useState } from 'preact/compat'
 import { useSnapshot } from 'valtio'
 import BlockchainPost from 'components/BlockchainList/BlockchainPost'
+import InfiniteScroll from 'react-infinite-scroll-component'
 import ListLoading from 'components/ListLoading'
 import NoPosts from 'components/BlockchainList/NoPosts'
 import PostStore from 'stores/PostStore'
+import classnames, { display, flexDirection, gap } from 'classnames/tailwind'
 import useScrollToAnchor from 'hooks/useScrollToAnchor'
 
+const scrollContainer = classnames(
+  display('flex'),
+  flexDirection('flex-col'),
+  gap('gap-y-4')
+)
+
 function BlockchainPostsListSuspended() {
-  const { selectedPosts, selectedToken } = useSnapshot(PostStore)
+  const { selectedPosts } = useSnapshot(PostStore)
   useScrollToAnchor()
 
-  return (
-    <>
-      {!selectedPosts.length ? (
-        <NoPosts />
-      ) : (
-        selectedPosts
-          .filter((post) =>
-            selectedToken ? post.derivativeAddress === selectedToken : post
-          )
-          .map((post) => (
-            <BlockchainPost
-              key={post.id}
-              id={Number(post.id)}
-              timestamp={Number(post.timestamp)}
-              text={post.post}
-              sender={post.sender}
-              derivativeAddress={post.derivativeAddress}
-            />
-          ))
-      )}
-    </>
+  const data = selectedPosts.filter((post) =>
```

Comment:
Can we rename this `data` to something else? We already have `data`, so not to mix, I suggest: `selectedPostsByToken` or something like this

Reply:
Renamed